tests/test_copy_async.py

```diff
@@ -120,6 +120,17 @@ async def test_rows(aconn, format):
     assert aconn.info.transaction_status == pq.TransactionStatus.INTRANS
 
 
+@pytest.mark.parametrize("format", pq.Format)
+async def test_set_types(aconn, format):
+    cur = aconn.cursor()
+    await ensure_table_async(cur, "id serial primary key, data jsonb, data2 bigint")
+    async with cur.copy(
+        f"copy copy_in (data, data2) from stdin (format {format.name})"
+    ) as copy:
+        copy.set_types(["jsonb", "bigint"])
+        await copy.write_row([{"foo": "bar"}, 123])
+
```

Comment:
It would be wise to fetch back the data to verify it was written correctly (sorry, I missed doing it in my initial MR). Can you add it please?

Reply:
Yupp, done.